src/analytics/Table.js

```diff
@@ -103,7 +103,9 @@ const AnalyticTable = ({ cards }) => {
                 {primaryLabels.map((primaryLabel) => (
                   <td>
                     {groups[primaryLabel][label] || 0}
-                    <span className="percent">{groups[primaryLabel][label] || 0 / cards.length}%</span>
+                    {label !== 'Total' && (
+                      <span className="percent">{(groups[primaryLabel][label] ?? 0) / cards.length}%</span>
```

Comment:
I still think this should divide by `groups][primaryLabel].Total` not `cards.length`. It also needs a to Fixed call.

Reply:
I think it makes more sense to show percentage from the total pool of cards

Reply:
You can always get the numbers you want by setting one sort to 'unsorted'